tcms/testplans/templates/testplans/get.html

```diff
@@ -84,12 +83,16 @@ <h2 class="card-pf-title">
         <div class="col-xs-12 col-sm-12 col-md-12">
             <div class="card-pf card-pf-accented">
                 <div class="card-pf-heading">
-                    <h2 class="card-pf-title">
+                    <div class="card-pf-title">
                         {% trans 'Confirmed test cases' %}
-                    </h2>
+                    </div>
                 </div>
                 <div class="card-pf-body">
-                    <div id="confirmed-testcases" class="list-group list-view-pf list-view-pf-view"></div>
+                    <div class="list-group list-view-pf list-view-pf-view">
+                        {% include 'include/tp_toolbar.html' %}
+                    </div>
```

Comment:
I don't see a compelling reason not to include the HTML directly in this template instead of making it another file.